beacon-chain/core/light-client/store.go

```diff
@@ -9,30 +9,30 @@ import (
 type Store struct {
 	mu sync.RWMutex
 
-	lastLCFinalityUpdate   interfaces.LightClientFinalityUpdate
-	lastLCOptimisticUpdate interfaces.LightClientOptimisticUpdate
+	lastFinalityUpdate   interfaces.LightClientFinalityUpdate
+	lastOptimisticUpdate interfaces.LightClientOptimisticUpdate
 }
 
-func (s *Store) SetLastLCFinalityUpdate(update interfaces.LightClientFinalityUpdate) {
+func (s *Store) SetLastFinalityUpdate(update interfaces.LightClientFinalityUpdate) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
-	s.lastLCFinalityUpdate = update
+	s.lastFinalityUpdate = update
 }
 
-func (s *Store) GetLastLCFinalityUpdate() interfaces.LightClientFinalityUpdate {
+func (s *Store) GetLastFinalityUpdate() interfaces.LightClientFinalityUpdate {
```

Comment:
One more comment: Our unwritten rule is to avoid `Get` in getters. So this should be just `LastFinalityUpdate`